test/configurator/ConfiguratorTest.cpp

```diff
@@ -0,0 +1,91 @@
+#include <k2/appbase/Appbase.h>
+#include <k2/common/Common.h>
+#include <k2/common/Timer.h>
+#include <k2/transport/PayloadSerialization.h>
+#include <k2/transport/Status.h>
+
+#include "k2/configurator/ConfiguratorDTO.h"
+
+namespace k2 {
+namespace log {
+inline thread_local k2::logging::Logger configuratortest("k2::configuratortest");
+}
+
+class ConfiguratorTest {
+    public:
+        seastar::future<> start() {
+
+            K2LOG_I(log::configuratortest, "Registering message handlers");
+            auto ep = RPC().getServerEndpoint(TCPRPCProtocol::proto);
+
+            _singleTimer.setCallback([ep, this] {
```

Comment:
I don't think you need a timer for this. You can assign the result from the make_ready_future call below to a member variable